contracts/mocks/MockBurnFactory.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.4.24;
+
+import "./MockRedemptionManager.sol";
+import "../modules/ModuleFactory.sol";
+import "../libraries/Util.sol";
+
+/**
+ * @title Mock Contract Not fit for production environment
+ */
+
+contract MockBurnFactory is ModuleFactory {
```

Comment:
I think we should inherit the main contract in the mock contract and overwrite the functions we need to. This will help the reader understand what exactly we are mocking and why. 
`contract MockBurnFactory is ModuleFactory` 
can be 
`contract MockBurnFactory is BurnFactory`

Same goes for all mock contracts.